src/RestApiContext.php

```diff
@@ -48,27 +48,21 @@ public function iSendARequestWithBody($method, $url, PyStringNode $body)
     }
 
     /**
-     * Sends HTTP request to specific URL with raw body from PyString.
-     *
-     * @param string    $method     request method
-     * @param string    $url        relative url
-     * @param TableNode $parameters
-     *
-     * @When /^(?:I )?send a ([A-Z]+) request to "([^"]+)" with parameters:$/
+     * @When I send a POST request to :url as HTML form with body:
      */
-    public function iSendARequestWithParameters($method, $url, TableNode $parameters = null)
+    public function iSendAPostRequestToAsHtmlFormWithBody($url, TableNode $body) 
     {
-        $this->restApiBrowser->sendRequest($method, $url, $parameters->getRowsHash());
-    }
+        $formElements = [];
+        foreach ($body as $element) {
 
-    /**
-     * @When I attach the following files:
-     */
-    public function iAttachTheFollowingFiles(TableNode $files)
-    {
-        foreach ($files as $file) {
-            $this->restApiBrowser->addFileToRequest($file['name'], $file['path']);
+            if (!isset($element['object'])) {
```

Comment:
This check should go in `Form` object to be sure we use always `Form` in the right way.